### DISCUSSION TITLE
How to output text at a specific location in trame?

### DISCUSSION THREAD

**Discussion by m2msoft** [Kitware/trame#186] [Q&A] [answered]
I don't know much about programming, so I ask too many questions. Thanks for the help.

You want to draw a point and put a text description at that location.

However, all text-related methods such as vtkTextActor and vtkFollower generate a No Serializer for ... error.

```
    actor = vtkTextActor()
    actor.SetInput(name)
    prop = actor.GetTextProperty()
    prop.SetBackgroundColor(0.0, 0.0, 0.0)
    prop.SetBackgroundOpacity(0.5)
    prop.SetFontSize(16)
    coord = actor.GetPositionCoordinate()
    coord.SetCoordinateSystemToWorld()
    coord.SetValue((pt[0], pt[1], pt[2]))
```

[Examples](https://discourse.vtk.org/uploads/default/original/1X/9b237fcf89a227790e8b85c5e54ecb6171e873f7.gif)

**Comment by jourdain** [comment, marked as answer]
As of yet, those are not supported with LocalView but if you switch to the RemoteView, they will work.

**Reply by m2msoft** [reply]
Thanks. I solved it.
